tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "forceConsistentCasingInFileNames": true,
     "strict": false,
     "strictNullChecks": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "declaration": true
```

Comment:
This `declaration` property is ultimately what will yield the Typescript declarations file to allow for out-the-box typing of this package.